Libplanet.Extensions.Cocona/Commands/Key/DerivationCommand.cs

```diff
@@ -0,0 +1,34 @@
+namespace Libplanet.Extensions.Cocona.Commands.Key
+{
+    using System;
+    using global::Cocona;
+    using Libplanet.Crypto;
+
+    public class DerivationCommand
+    {
+        public void PrivateKey(
+            [Argument("PRIVATE-KEY")]
```

Comment:
It would be better if it has a description:

```suggestion
            [Argument(
                "PRIVATE-KEY",
                Description = "A private key that derives the address/public key from."
            )]
```